app/controllers/presentations_controller.rb

```diff
@@ -1,8 +1,16 @@
 class PresentationsController < ApplicationController
   def index
-    @presentations = Presentation.joins(:event)
-                                 .merge(Event.order(date: :desc))
-                                 .page(params[:page])
+    if params[:presentation] && params[:presentation][:person_id].present?
```

Comment:
Brendan has asked me to move to the safe navigation operator. 

http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/